bin/windows/plantuml-documentation-petstore.bat

```diff
@@ -5,6 +5,6 @@ If Not Exist %executable% (
 )
 
 REM set JAVA_OPTS=%JAVA_OPTS% -Xmx1024M -DloggerPath=conf/log4j.properties
-set ags=generate  --artifact-id "plantuml-petstore-documentation" -i modules\openapi-generator\src\test\resources\2_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
+set ags=generate -t modules\openapi-generator\src\main\resources\plantuml-documentation -i modules\openapi-generator\src\test\resources\3_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
```

Comment:
Think you need to remove the `-documentation` suffix from this file too.
```suggestion
set ags=generate -t modules\openapi-generator\src\main\resources\plantuml -i modules\openapi-generator\src\test\resources\3_0\petstore.yaml -g plantuml -o samples\documentation\petstore\plantuml
```

Reply:
Fixed via https://github.com/OpenAPITools/openapi-generator/pull/6298/commits/851b94a3c800e69fc374797a024236ccd06b2236